Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -265,9 +292,9 @@ open class MessageContentCell: MessageCollectionViewCell {
         messageTopLabel.frame = CGRect(origin: origin, size: attributes.messageTopLabelSize)
     }
 
-    /// Positions the cell's bottom label.
+    /// Positions the message bubble's bottom label.
     /// - attributes: The `MessagesCollectionViewLayoutAttributes` for the cell.
-    open func layoutBottomLabel(with attributes: MessagesCollectionViewLayoutAttributes) {
+    open func layoutMessageBottomLabel(with attributes: MessagesCollectionViewLayoutAttributes) {
```

Comment:
This is an API breaking change. Let's make sure to add a CHANGELOG entry for this. Please refer to the other breaking change entries and follow that format.